cmd/controller/velerovolume/config/config.go

```diff
@@ -12,6 +12,7 @@ type VeleroVolumeCfg struct {
 	ExcludeNamespaces  string `yaml:"excludeNamespaces,omitempty"`
 	IncludeVolumeTypes string `yaml:"includeVolumeTypes,omitempty"`
 	ExcludeVolumeTypes string `yaml:"excludeVolumeTypes,omitempty"`
+	ExcludeJobs        bool   `yaml:"excludeJobs"`
```

Comment:
ExcludeJobs should be list of excluding job names

Reply:
I don't think that it is a good idea.
First I would like to be able to ignore all jobs, ignoring their names and count (some jobs can be later deployed in the cluster and I would like that the controller continue to ignore the new jobs without reconfiguring it all the times). Jobs are often used to update volumes used by another pods, so backup can be done from these pods.
Then jobs can be created by cronjob, so we can not know their name, as each time a new job is scheduled, a new name is assigned to it.